src/main/java/net/sf/jabref/model/entry/BibEntry.java

```diff
@@ -194,7 +202,6 @@ public void setType(EntryType type) {
     /**
      * Returns the contents of the given field, or null if it is not set.
      */
-    @Deprecated //Use getFieldOptional instead
```

Comment:
Right, I can do that, I took `Object` casue `String.valueOf(...)` is also accepting that.